### ISSUE TITLE
After following the UE5 steps to enable, we need to compile everything whenever we generate project

### ISSUE THREAD

**Issue by NG-Joel** [westquote/SquidTasks#9]
After right-clicking on the <project>.uproject file and selecting "Generate Visual Studio project files" we need to recompile everything. This makes it very slow to work with since you might generate projects multiple times per day.

1. Compile DebugGameEditor|Win64 and launch (to ensure that everything is compiled)
2. Close Visual Studio
3. Open VS again and launch. Notice that everything is compiled and it launches instantly
4. Close Visual Studio
5. Right-click <project>.uproject file and select "Generate Visual Studio project files" 
6. Open visual studio and launch. Notice that it will now compile everything

I noticed that in step 5 all the "Definition.*.h" files would be generated with "#define WITH_CPP_COROUTINES 0", but after we compiled through Visual Studio it would be set to "#define WITH_CPP_COROUTINES 1" (since we want Coroutines). 
Our fix for this was to add "bEnableCppCoroutinesForEvaluation = true;" into UnrealEditor.Target.cs and now they generate with 1 instead of 0.
It would be good to update the setup guide with this information and if you know a better way of doing it to include that instead.

**Comment by westquote** [comment]
Hello, and thanks for taking the time to explain this issue. Near the end of Step 1 of our [UE5 setup guide](https://github.com/westquote/SquidTasks/blob/main/documentation/UE5IntegrationGuide.md), we instruct the user to insert "bEnableCppCoroutinesForEvaluation = true;" into their Target file. It looks like you are adding it to the UnrealEditor.Target.cs file - is that a distinction you'd like clarified in the setup guide?  Let me know!

**Comment by NG-Joel** [comment]
> Hello, and thanks for taking the time to explain this issue. Near the end of Step 1 of our [UE5 setup guide](https://github.com/westquote/SquidTasks/blob/main/documentation/UE5IntegrationGuide.md), we instruct the user to insert "bEnableCppCoroutinesForEvaluation = true;" into their Target file. It looks like you are adding it to the UnrealEditor.Target.cs file - is that a distinction you'd like clarified in the setup guide? Let me know!

Yes, exactly. We are adding it to both UnrealEditor.Target.cs and <MyProject>.target.cs. Just having it in <MyProject>.target.cs made us having to recompile the entire engine every time we generated project files. My work-around saves us a lot of time, so I hope it can save others lots of time also.
